src/resources/Catalogs/Interfaces.ts

```diff
@@ -0,0 +1,30 @@
+export interface CatalogsListOptions {
```

Comment:
I think this goes hand-in-hand with the `PageModel`, so I think I would put a name similar to it, like `CatalogsPageOptions`

Reply:
Yes. The reason why I used the `List` term is that it goes hand-in-hand with the `catalog.list` method.

Reply:
👍 OK that's good